devlog/_plan/260902_cursor_bundle_effort_table/040_wp4_gui_provenance_hint.md

```
@@ -0,0 +1,101 @@
+### MODIFY `gui/src/pages/integrations/CursorIntegrationPage.tsx`
+
+1. Under the "What Cursor will show" heading (line ~141), replace the static hint paragraph with a
+   provenance line:
+```tsx
+<p className="muted">
+  {status.effortTable.source === "bundle"
+    ? t("integrations.cursor.ladderFromBundle", { version: status.effortTable.version ?? "?" })
+    : t("integrations.cursor.ladderFromStatic")}
+</p>
+```
+2. Reasoning cell (line ~155): when `model.reasoning` is null render
```

markdownlint-cli2 (0.23.2)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)